evap/evaluation/tools.py

```diff
@@ -276,7 +277,11 @@ def next_row(self) -> None:
         self.cur_col = 0
         self.cur_row += 1
 
-    def write_row(self, vals: Iterable[str], style: str = "default") -> None:
+    def write_row(
+        self,
+        vals: Iterable[OptionalStrOrOptionalFloat],
```

Comment:
This has already been typed in #2073 (and I prefer the version from that). Could you rebase your branch to incorporate the changes?